chapter02/worksheets/solution2.2.6.1.scala

```diff
@@ -18,11 +21,16 @@ def fromPairs[A](xs: Seq[(A, A)]): Seq[A] = {
 
 val result = fromPairs(List(("a","b"), ("c","<nothing>")))
 val expected = List("a", "b", "c", "<nothing>")
-
 assert(result == expected)
 
+val a: Seq[Int] = this.fromPairs(List((1, 2), (3, 4)))
```

Comment:
Here is another `this.fromPairs` - maybe remove it?

Reply:
Thanks - 'grep' seems to be on strike here today :)